app/models/service_template_ansible_playbook.rb

```diff
@@ -52,6 +52,15 @@ def self.create_catalog_item(options, auth_user)
     end
   end
 
+  def self.manager_valid?(manager)
```

Comment:
@bzwei - I found `any?` does the same thing when called from an ActiveRecord Proxy - so went that direction.